src/redismodule.h

```diff
@@ -46,6 +46,10 @@
 #define REDISMODULE_REPLY_INTEGER 2
 #define REDISMODULE_REPLY_ARRAY 3
 #define REDISMODULE_REPLY_NULL 4
+#define REDISMODULE_REPLY_MAP 5
+#define REDISMODULE_REPLY_SET 6
+#define REDISMODULE_REPLY_BOOL 7
+#define REDISMODULE_REPLY_DOUBLE 7
```

Comment:
if we add the other types (bignum, verbatim), we'll want to add them to the module API too..

Reply:
Done